tests/test_visitors/test_ast/test_classes/test_base_classes/test_base_builtin_classes.py

```diff
@@ -11,6 +11,15 @@
 enum_with_primitive2 = 'class First({0}, Enum): ...'
 enum_with_primitive3 = 'class First({0}, EnumMeta): ...'
 enum_with_primitive4 = 'class First({0}, enum.EnumType): ...'
+enum_with_primitive5 = 'class First({0}, ReprEnum): ...'
+enum_with_primitive6 = 'class First({0}, enum.ReprEnum): ...'
+
+concrete_enum_with_primitive1 = 'class First({0}, enum.StrEnum): ...'
+concrete_enum_with_primitive2 = 'class First({0}, StrEnum): ...'
+concrete_enum_with_primitive3 = 'class First({0}, IntEnum): ...'
+concrete_enum_with_primitive4 = 'class First({0}, enum.IntEnum): ...'
+concrete_enum_with_primitive5 = 'class First({0}, IntFlag): ...'
+concrete_enum_with_primitive6 = 'class First({0}, enum.IntFlag): ...'
```

Comment:
Please, also test that `class First(PRIMITIVE, TextChoices): ...'` and similar is not possible